### ISSUE TITLE
Help about Winding classes

### ISSUE THREAD

**Issue by cjw1254** [Eomys/pyleecan#530]
Hello. It's my first time using the pyleecan so I'm following the tutorials first. (Tutorials: https://github.com/Eomys/pyleecan/blob/master/Tutorials/tuto_Machine.ipynb)

While I m following the first tutorial, the result is the same as answer
![1](https://user-images.githubusercontent.com/103762228/163587509-6c95551f-88eb-49ea-996f-d769df4722e9.PNG)

Also, there r another issue on 2nd tutorial as follows, and I think there r no Winding classes for now.

![2](https://user-images.githubusercontent.com/103762228/163588552-35209f89-b405-47e7-b224-b256750f3c35.PNG)
![3](https://user-images.githubusercontent.com/103762228/163588568-bba58861-eb13-4636-bec7-96db9326a450.PNG)

I ve tried another winding class 'WindingCW2LR', but error was the same as above.

Is it a problem with version?? or can I add winding classes by myself??

I'm using python ver. 3.7.13 with anaconda & VS code
Also Pyleecan info is as follow:
Running date: April 15, 2022
Pyleecan version:1.3.8
SciDataTool version:2.3.3

Thx a lot for superb SW and I hope I can contribute someday.

**Comment by BonneelP** [comment]
Hello, 
Thank you for your interest in pyleecan.
Sadly the tutorial are not up to date :/ I should have time to work on that on next week.
By waiting WindingCW2LT doesn't exist any more, now there is only one class named "Winding" where you can set coil_pitch, qs, p, Nlayer and the corresponding optimal connection matrix is computed thanks to star of slot method from swat-em. Normally all that can be done from the GUI. 

Best regards,
Pierre

**Comment by cjw1254** [comment]
Oh I see, you mean the class "Winding" documentated as follow?
: https://www.pyleecan.org/pyleecan.Classes.Winding.html
I m trying to work on it with language not GUI so I ll follow the documentation above first.
If it allows ur time, plz let me know when the tutorials done.
Appreciate ur kind reply.

**Comment by BonneelP** [comment]
Yes it's this class. You can also find the csv description here https://github.com/Eomys/pyleecan/blob/master/pyleecan/Generator/ClassesRef/Machine/Winding.csv 
You can also take inspiration from the machine library: https://github.com/Eomys/pyleecan/tree/master/pyleecan/Data/Machine
open any machine json and search for "winding", for instance Prius: 
![image](https://user-images.githubusercontent.com/20533034/163963110-e3791dd0-d725-402a-88f8-c7f7db6d8ac8.png)
![image](https://user-images.githubusercontent.com/20533034/163963156-9f7bcc8f-f309-4997-bede-5c98b25ce360.png)

Best regards,
Pierre

**Comment by BonneelP** [comment]
Hello,
It took me longer that expected but the tutorials are now up to date, same for the website. 
Do you need any further assistance or can we close this issue ?
Best regards,
Pierre
